src/Features/Core/Portable/AddImport/CodeActions/InstallPackageAndAddImportCodeAction.cs

```diff
@@ -92,14 +73,12 @@ private class InstallPackageAndAddImportCodeAction : CodeAction
                 protected override async Task<IEnumerable<CodeActionOperation>> ComputeOperationsAsync(
                     CancellationToken cancellationToken)
                 {
-                    var installData = await _installData.GetValueAsync(cancellationToken).ConfigureAwait(false);
-
-                    var oldText = await installData.OldDocument.GetTextAsync(cancellationToken).ConfigureAwait(false);
-                    var newText = await installData.NewDocument.GetTextAsync(cancellationToken).ConfigureAwait(false);
+                    var oldText = await OriginalDocument.GetTextAsync(cancellationToken).ConfigureAwait(false);
+                    var newText = oldText.WithChanges(TextChanges);
```

Comment:
Any reason not to just use `GetUpdatedDocumentAsync()` (and make `TextChanges` private instead of protected)?